x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -389,7 +416,7 @@ export const signalRulesAlertType = ({
           }
         } else {
           const errorMessage = buildRuleMessage(
-            'Bulk Indexing of signals failed. Check logs for further details.'
+            `Bulk Indexing of signals failed. ${result.errorMessage} Check logs for further details.`
```

Comment:
Adds the error message caught from `searchAfterBulkCreate`. This will help provide more context for any failures that appear in the "recent failures" tab in the rule details page.